avanza/models/market.go

```diff
@@ -4,6 +4,26 @@ type GetMarketDataParams struct {
 	OrderBookID string `path:"orderbookID" validate:"required"`
 }
 
+type GetChartDataResponse struct {
+	Ohlc []struct {
```

Comment:
Prefer `Ohlc` to be uppercase
```suggestion
	OHLC []struct {
```